data/events/2024/curitiba/main.yml

```
@@ -53,35 +53,41 @@ team_members: # Name is the only required field for team members.
   - name: "Alex Mazur"
     employer: "ThoughtWorks"
     linkedin: "https://www.linkedin.com/in/alex-mazur"
+    image: "alex.jpeg"	
   - name: "Marcelo Baptista"
     employer: "C6 Bank"
     linkedin: "https://www.linkedin.com/in/marcelocbaptista"
-  - name: "Emerson Saito"
+    image: "marcelo.jpeg"	
+  - name: "Emerson Sachio Saito"
     employer: "SERPRO"
     linkedin: "https://www.linkedin.com/in/emerson-sachio-saito"
-  - name: "Fabiano"
-    #employer: "Acme Anvil Co."
-    #linkedin:
+    image: "emerson.png"
+  - name: "Fabiano Sardenberg Kuss"
+    employer: "SERPRO"
+    linkedin: "https://www.linkedin.com/in/fabiano-kuss-37742956/"
+    image: "fabiano.jpeg"
     #twitter: "devopsdays"
   - name: "Pedro Alves Batista"
     #employer: "Acme Anvil Co."
     linkedin: "https://www.linkedin.com/in/pedro-alves-batistahq3"
+    image: "pedro.jpeg"
     #github: "devopsdays"
     #facebook: "https://www.facebook.com/sally.fields"
     #linkedin: 
     #website: "https://mattstratton.com"
     #image: "sally-fields.jpg"
 
-organizer_email: "curitiba@devopsdays.org" # Put your organizer email address here
+organizer_email: "devopsdayscuritiba@gmail.com" # Put your organizer email address here
```

As per [the organizers guide](https://devopsdays.org/organizing#initial-steps-for-organizing), we require that your organiser email be your city email. 

```suggestion
organizer_email: "curitiba@devopsdays.org" # Put your organizer email address here
```

---

No one from the local (Curitiba) organization knows how to access this email: curitiba@devopsdays.org

---

Thanks for the update. If you require assistance with anything with your event, email info@ (if you used your city email, the global core organisers would also get a copy of your communications and can help more proactively).

---

@esaito Hello. curitiba@devopsdays.org is a distribution list, which means that any email sent to that address is sent to the members of the list directly. There's nothing to access. Try sending an email to that list yourself and seeing what happens; the other members of your local team will see it in their inboxes.